src/schema/ecommerce/types/offer.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  GraphQLID,
+  GraphQLObjectType,
+  GraphQLString,
+  GraphQLInt,
+} from "graphql"
+import { connectionDefinitions } from "graphql-relay"
+import date from "schema/fields/date"
+import { OrderPartyUnionType } from "./order_party_union"
+import { OrderType } from "./order"
+export const OfferType = new GraphQLObjectType({
+  name: "Offer",
+  fields: () => ({
+    id: {
+      type: GraphQLID,
+      description: "ID of the offer",
+    },
+    createdAt: date,
+    creatorId: {
+      type: GraphQLString,
+      description: "Id of the user who created the order",
```

Comment:
i think we can resolve this to a `user`

Reply:
Do we have to be careful about auth here? I assume e.g. we don't always want partners to get the buyers' emails.

Reply:
good question, i don't think we need to be worried in this specific one since this info user data is also in `Order.from`. The part that i think we should be more careful is to not expose partner's contacts to buyers. In the end we may want to make this field a `seller_only` field in Exchange.

Reply:
Ok we can iterate on that :+1: